cryptography/src/darwinTest/kotlin/com/wire/kalium/cryptography/BaseProteusClientTest.kt

```diff
@@ -14,7 +16,7 @@ actual open class BaseProteusClientTest actual constructor() {
         proteusStore: ProteusStoreRef,
         databaseKey: ProteusDBSecret?
     ): ProteusClient {
-        return ProteusClientImpl(proteusStore.value, ProteusDBSecret("secret"))
+        return ProteusClientImpl(proteusStore.value, defaultContext = standardScope, ioContext = standardScope)
```

Comment:
This you lost the `ProteusDBSecret("secret")`. Darwin only supports encrypted proteus storage so the tests will always fail now.